scripts/api_backend_dev.sh

```diff
@@ -12,7 +12,8 @@ if [[ $1 == "--e2e" ]]; then
   previous_file="./dev/config.previous.yaml"
   config_file="./dev/config.yaml"
   cp ${config_file} ${previous_file}
-  sed 's/enable_auth: true/enable_auth: false/g' ${previous_file} >${config_file}
+  #TODO(OIDC): we need to reproduce old "enable_auth: false" by replacing providers value by [] ?
+  #sed 's/enable_auth: true/enable_auth: false/g' ${previous_file} >${config_file}
```

Comment:
I think it's simpler to keep the attribute `enable_auth` since it's widely used to activate or deactivate things across the API and the UI.

Reply:
I agree